File: C++/Mlecz.cpp
#include "Mlecz.h"

Mlecz::Mlecz(Swiat& swiat, int x, int y) :
	Roslina(swiat, x, y)
{
	this->znak = MLECZ;
}

Mlecz::~Mlecz()
{
}

bool Mlecz::akcja()
{
	this->rozmnoz(this->polozenie.x, this->polozenie.y);
	this->rozmnoz(this->polozenie.x, this->polozenie.y);
	return true;
}

File: Java/Mlecz.java
import java.awt.*;

public class Mlecz extends Roslina{
    public Mlecz(Swiat swiat, int x, int y) {
        super(swiat, x, y);
        this.znak = TypOrganizmu.MLECZ;
        setKolor(Color.YELLOW);
    }
    boolean akcja()
    {
        this.rozmnoz(this.x, this.y);
        this.rozmnoz(this.x, this.y);
        return true;
    }
}
